### PR TITLE
Add details to contributors list

### PR DESCRIPTION
**Pull request by iykat**
#DataUmbrellaPyMCSprint

:books: Documentation preview en :books:: https://pymc-data-umbrella--224.org.readthedocs.build/en/224/

:books: Documentation preview es :books:: https://pymc-data-umbrella-es--224.org.readthedocs.build/es/224/

:books: Documentation preview pt :books:: https://pymc-data-umbrella-pt--224.org.readthedocs.build/pt/224/

### CODE REVIEW

File: 2023-03_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -111,3 +111,16 @@ Brooklyn, New York
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
 ::::
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="https://raw.githubusercontent.com/data-umbrella/data-umbrella.github.io/main/images/logo/umb-trans.png" class="rounded-circle">
+
```
Comment (reshamas) [inline 2023-03_sprint/sprint_parties/contributors.md:118]: The `src` link to image needs to be updated.
This is the preview:
https://pymc-data-umbrella--224.org.readthedocs.build/en/224/2023-03_sprint/sprint_parties/contributors.html

If you prefer not to have a photo, try this one:
```
src="../../_static/contributors/blank_person.jpg"
```

File: 2023-03_sprint/sprint_parties/contributors.md [hunk 2]
```diff
@@ -111,3 +111,16 @@ Brooklyn, New York
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
 ::::
```
Comment (OriolAbril) [inline 2023-03_sprint/sprint_parties/contributors.md:113]: Also the grid item needs to be _inside_ the grid directive. That is, within the `::::{grid}` opening and the `::::` closing in this line (note the difference between this 4 colon fences and the innner ones of 3 colons only for each grid item)

Comment (OriolAbril) [reply 2023-03_sprint/sprint_parties/contributors.md:113]: You can see in the preview how your info is outside the grid: https://pymc-data-umbrella--224.org.readthedocs.build/en/224/2023-03_sprint/sprint_parties/contributors.html

Comment (iykat) [reply 2023-03_sprint/sprint_parties/contributors.md]: @OriolAbril I have updated the grid item positioning.

File: 2023-03_sprint/sprint_parties/contributors.md (hunk 1 above)
Comment (iykat) [reply 2023-03_sprint/sprint_parties/contributors.md]: @reshamas I have updated the image url

File: 2023-03_sprint/sprint_parties/contributors.md [hunk 3]
```diff
@@ -110,4 +110,16 @@ Brooklyn, New York
 [{fab}`linkedin`](https://www.linkedin.com/in/saitejaswie/)
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="https://media.licdn.com/dms/image/C4D03AQECXsfatPDn7Q/profile-displayphoto-shrink_800_800/0/1638788712245?e=1685577600&v=beta&t=LXFC6Qlo6ZxSOWjO5ryo7hTfnd3M2dJM5KXVeyE2KeA" class="rounded-circle">
```
Comment (OriolAbril) [inline 2023-03_sprint/sprint_parties/contributors.md:116]:
```suggestion
<img alt="" src="https://media.licdn.com/dms/image/C4D03AQECXsfatPDn7Q/profile-displayphoto-shrink_800_800/0/1638788712245?e=1685577600&v=beta&t=LXFC6Qlo6ZxSOWjO5ryo7hTfnd3M2dJM5KXVeyE2KeA" class="rounded-circle">
```

see also https://github.com/pymc-devs/pymc-data-umbrella/issues/226, trying to make things a bit easier for future us.

**Pull request merged**